content/agent/basic_agent_usage/heroku.md

```diff
@@ -43,14 +43,22 @@ In addition to the environment variables shown above, there are a number of othe
-| `DD_TAGS` | *Optional.* Sets additional tags provided as a comma-delimited string. For example, `heroku config:set DD_TAGS=simple-tag-0,tag-key-1:tag-value-1`. The buildpack automatically adds the tags `dyno` and `dynohost` which represent the Dyno name (e.g. web.1) and host ID (e.g. 33f232db-7fa7-461e-b623-18e60944f44f) respectively. See the ["Guide to tagging"][3] for more information. |
+| `DD_DYNO_HOST` | *Optional.* Set to `true` to use the dyno name (e.g. `web.1` or `run.1234`) as the hostname. See the [hostname section](#hostname) below for more information. Defaults to `false`. |
+| `DD_TAGS` | *Optional.* Sets additional tags provided as a comma-delimited string. For example, `heroku config:set DD_TAGS=simple-tag-0,tag-key-1:tag-value-1`. The buildpack automatically adds the tags `dyno` (the dyno name, e.g. `web.1`), `dynohost` (the host machine ID, e.g. `33f232db-7fa7-461e-b623-18e60944f44f`), and `dynotype` (the type of dyno, e.g `run` or `web`). See the ["Guide to tagging"][3] for more information. |
 | `DD_HISTOGRAM_PERCENTILES` | *Optional.* Optionally set additional percentiles for your histogram metrics. See [How to grpah percentiles][4]. |
 | `DISABLE_DATADOG_AGENT` | *Optional.* When set, the Datadog agent will not be run. |
 | `DD_APM_ENABLED` | *Optional.* The Datadog Trace Agent (APM) is run by default. Set this to `false` to disable the Trace Agent. |
 | `DD_AGENT_VERSION` | *Optional.* By default, the buildpack installs the latest version of the Datadog Agent available in the package repository. Use this variable to install older versions of the Datadog Agent (note that not all versions of the Agent may be available). |
 | `DD_SERVICE_ENV` | *Optional.* The Datadog Agent automatically tries to identify your environment by searching for a tag in the form `env:<environment name>`. For more information, see the [Datadog Tracing environments page][5]. |
 
+## Hostname
+
+Heroku dynos are ephemeral—they can move to different host machines whenever new code is deployed, configuration changes are made, or resouce needs/availability changes. This makes Heroku flexible and responsive, but can potentially lead to a high number of reported hosts in Datadog. Datadog bills on a per-host basis, and the buildpack default is to report actual hosts, which can lead to higher than expected costs.
+
+Depending on your use case, you may want to set your hostname so that hosts are aggregated and report a lower number.  To do this, Set `DD_DYNO_HOST` to `true`. This will cause the Agent to report the hostname as the dyno name (e.g. `web.1` or `run.1234`) and your host count will closely match your dyno usage. One drawback is that you may see some metrics continuity errors whenever a dyno is cycled.
```

Comment:
r/ `This will cause the Agent to report the hostname as the dyno name (e.g. `web.1` or `run.1234`) and your host count will closely match your dyno usage.` / 

`This causes the Agent to report the hostname as the dyno name (e.g. `web.1` or `run.1234`) in order to match your host count with your dyno usage.`

Reply:
actually, simplify a bit further: The Agent will report the hostname as....

Reply:
hey @jyee couple comments here